sample/src/com/vungle/publisherSampleAndroid/MainActivity.java

```diff
@@ -101,6 +105,34 @@ public void run() {
 		}
 	};
 
+	private final EventListener vungleSecondListener = new EventListener() {
+		@Override
+		public void onVideoView(boolean isCompletedView, int watchedMillis, int videoDurationMillis) {
+			// Called each time a video completes.  isCompletedView is true if >= 80% of the video was watched.
+		}
+
+		@Override
+		public void onAdStart() {
+			// Called before playing an ad.
+		}
+
+		@Override
+		public void onAdUnavailable(String reason) {
+			// Called when VunglePub.playAd() was called but no ad is available to show to the user.
+		}
+
+		@Override
+		public void onAdEnd(boolean wasCallToActionClicked) {
+			// Called when the user leaves the ad and control is returned to your application.
```

Comment:
We can get rid of all of the duplicate commenting, it's noted in the original event listener so we don't need it to be added in a second time